core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -237,8 +238,8 @@ contract Liquidatable is PricelessPositionManager {
     function dispute(uint id, address sponsor) external disputable(id, sponsor) fees() {
         LiquidationData storage disputedLiquidation = _getLiquidationData(sponsor, id);
 
-        FixedPoint.Unsigned memory disputeBondAmount = _getCollateral(disputedLiquidation.rawLockedCollateral).mul(
-            disputeBondPct
+        FixedPoint.Unsigned memory disputeBondAmount = disputedLiquidation.lockedCollateral.mul(disputeBondPct).mul(
+            _getCollateral(disputedLiquidation.rawFeeAttenuation)
```

Comment:
Can you explain the math here? `_getCollateral(x)` returns `x * feeMultiplier`, while `rawFeeAttentuation` should be `1 / feeMultiplier` so doesn't this just return 1?

Reply:
The fee multiplier changes over time to take into account the fees, so a more accurate representation would be:
```
rawFeeAttenuation = 1 / feeMultiplier_t_0
... time passes ...
feeAttenuation = rawFeeAttenuation * feeMultiplier_t_1
```

Reply:
If no fees had been charged between t_0 and t_1, this would result in feeAttenuation == 0, but if some fees had been charged, feeAttenuation < 0.

Reply:
As discussed IRL, it goes down slightly when fees are charged over the course of a liquidation.